# Define the goal
string tfName;  # Which object we want to find
---
# Define the result
int8 status #1: there; 0: dead
---
# Define a feedback message
float32 distance_remaining